rezasm-source/rezasm-core/src/simulation/transform/transformable.rs

```diff
@@ -41,6 +47,26 @@ impl Transformable {
                 simulator.get_memory_mut().write(address.clone(), &data)
             }
             Transformable::FileReadTransformable(cursor) => todo!(), //must be todo until read instructions are done
+            _ => Ok(()),
```

Comment:
no

Reply:
This fallthrough case is a bad idea. Any unhanded case should raise a compilation error.